resources/Linux/raspberrypi/iothub_convenience_sample.c

```diff
@@ -0,0 +1,314 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Why do wen need the sample here?

Reply:
It is just a iothub-related sample for RPI(Linux device). Like the  emptysketch.ino for Arduino device DevKit. The file is used as a scaffolded sample file when user create a project of RPI.
We will change this sample file to a more general file, like `iothub_sample.c`.

Reply:
[TODO] 
1. Use a simpler scaffold file.: [iothub_ll_telemetry_sample.c](https://github.com/Azure/azure-iot-sdk-c/blob/master/iothub_client/samples/iothub_ll_telemetry_sample/iothub_ll_telemetry_sample.c)
2. Use URL to get template files instead of storing files under `resource` folder.